source/common/http/http2/codec_impl.h

```diff
@@ -317,9 +301,11 @@ class ConnectionImpl : public virtual Connection,
     bool pending_send_buffer_high_watermark_called_ : 1;
     bool reset_due_to_messaging_error_ : 1;
     absl::string_view details_;
-    // See HttpConnectionManager.stream_idle_timeout.
-    std::chrono::milliseconds stream_idle_timeout_{};
-    Event::TimerPtr stream_idle_timer_;
+
+  protected:
```

Comment:
ditto on commenting the override.
alternately I don't see a reason to not have it public and group it - your call.